### DISCUSSION TITLE
We are making a Dolby Atmos video and plan to play it on Screen app.

### DISCUSSION THREAD

**Discussion by 771058380** [videovillage/community#163] [Q&A]
Hi Greg,

We are a team based in Beijing, China. We are making a Dolby Atmos video and plan to play it on Screen app. 

Would you please let us know if Screen app supports Dolby Atmos videos, and can recognize & play our video without problem?  

Look forward to hearing from you.

Best Regards,

**Comment by gregcotten** [comment]
> Hi Greg,
> 
> 
> 
> We are a team based in Beijing, China. We are making a Dolby Atmos video and plan to play it on Screen app. 
> 
> 
> 
> Would you please let us know if Screen app supports Dolby Atmos videos, and can recognize & play our video without problem?  
> 
> 
> 
> Look forward to hearing from you.
> 
> 
> 
> Best Regards,
> 
> 

I have no test media with Dolby Atmos encoding, but if QuickTime can play it Screen most likely can.
